site/_posts/2020-08-26-announcing-fusionauth-1-19.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: blog-post
+title: Announcing FusionAuth 1.19
+description: The FusionAuth 1.19 release offers stateless admin pages, multi-tenant SSO, more flexible configuration, and more.
+author: Dan Moore
+image: blogs/release-1-19/fusionauth-release-1-19.png
+category: blog
+excerpt_separator: "<!--more-->"
+---
+
+We're excited to announce the release of version 1.19 of FusionAuth, which shipped at the end of August. This version delivers new features and resolves issues for users running any older version.
+
+<!--more-->
+
+## Highlights
+
+In addition to bug fixes and user interface improvements, there are a few highlights we'd like to, well, highlight.
+
+### Revamped configuration options
+
+Configuration options are now consistent no matter how you set them. Every option can be specified in one of three ways:
+
+* a key value pair in the `fusionauth.properties` file
+* environment variables
+* Java system properties, set with the `-D` command line option when starting FusionAuth
+
+If you are running in Docker or Kubernetes, you'll have all the configuration flexibility without mounting any volumes.
+
+And if you are worried about your configuration breaking with these new settings, don't. All previous configuration options are backwards compatible. We will, however, nag you to update your configuration values or environment variables, because that’s how we roll. Learn more in the [configuration reference docs](/docs/v1/tech/reference/configuration).
```

Comment:
nag- maybe change to "kindly remind you"

Reply:
or... 

> We will, however, ~nag~ "kindly remind you" you to update your configuration.